docs/cosmos/swagger.go

```diff
@@ -0,0 +1,67 @@
+package docs
+
+import (
+	"github.com/shapeshift/unchained-cosmos/service"
+)
+
+// swagger:route GET /account/{pubkey} account-tag accountEndpointId
```

Comment:
These can live anywhere that is a child of the main.go.

It could be placed next to each endpoint but the example I was looking at did it this way